Library/Formula/llvm.rb

```diff
@@ -116,18 +149,49 @@ def install
     # Apple's libstdc++ is too old to build LLVM
     ENV.libcxx if ENV.compiler == :clang
 
-    if build.with?("lldb") && build.without?("clang")
-      raise "Building LLDB needs Clang support library."
-    end
+    (buildpath/"tools/clang").install resource("clang") if build.with? "clang"
 
-    if build.with? "clang"
-      (buildpath/"projects/libcxx").install resource("libcxx")
-      (buildpath/"tools/clang").install resource("clang")
-      (buildpath/"tools/clang/tools/extra").install resource("clang-tools-extra")
+    if build.with? "clang-extra-tools"
+      odie "clang's extra tools must be built with clang" if build.without? "clang"
+      (buildpath/"tools/clang/tools/extra").install resource("clang-extra-tools")
     end
 
+    (buildpath/"projects/libcxx").install resource("libcxx") if build.with? "libcxx"
+    (buildpath/"projects/libcxxabi").install resource("libcxxabi") if build.with? "libcxx"
```

Comment:
Because both lines use the same condition, it is better to use an `if` block here. (But thanks for reviewing and updating all the other instances where a trailing `if` modifier is better!)